### PR TITLE
Implemented sampling with weights with test cases, and added global v…

### PR DESCRIPTION
**Pull request by DavidLBick**
…ariable for the number of individuals in each generation

### CODE REVIEW

File: sampleParents.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    prefixSums, totalSum = getPrefixSums(fitnesses)
+
+    samples = np.zeros((NUM_INDIVIDUALS,))
+    for i in range(NUM_INDIVIDUALS):
+        samples[i] = singleSample(prefixSums, totalSum)
+
+    return samples
+
+
+def test_sampling():
```
Comment (aoshiro10) [inline sampleParents.py]: Move test to a different file.

**Review by aoshiro10 — CHANGES_REQUESTED** [review]
I think it looks pretty good overall. 

Just need to change tests to a separate file.

**Comment by DavidLBick** [comment]
Fixed

File: sampleParents.py (hunk 1 above)
Comment (DavidLBick) [reply sampleParents.py]: fixed

**Review by aoshiro10 — APPROVED** [review]
Looks good!

**Pull request merged**